.github/workflows/ci.yml

```diff
@@ -45,8 +57,8 @@ jobs:
           always-auth: true
           cache: 'yarn'
 
-      - name: Set version
-        if: github.repository_owner == 'Open-CMSIS-Pack'
+      - name: Set version (Linux/macOS)
+        if: runner.os == 'Linux' && github.repository_owner == 'Open-CMSIS-Pack'
```

Comment:
Is the step-name still accurate? I believe this condition doesn't include macOS.

Reply:
[Addressed review comments](https://github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/pull/408/commits/5f5e2b639c25d227b81ee445edb07880f2f2521b)